### PR TITLE
doc(README): refresh the doc structure

### LINKED ISSUE #215: Reorganize the documents content (including README) to reflect the latest changes
- [x] Overview
- [x] Operator features
- [x] Future features
- [ ] Getting Started (tutorial)
- [x] Compatibility (K8s, cm and Harbor)
- [ ] Documentations
  - [x] How does it work (architecture)
  - [x] Installation
  - [x] CRD references
  - [x] Customize images
  - [ ] Day2 configuration
- [x] Community
- [x] Contributions  (Mention contributors)
- [x] Recognition (mention OVH initialize this project)
- [x] License

### PR DESCRIPTION
**Pull request by steven-zou**
- update README
- update MAINTAINER
- create placeholder MD files
- move MD files to the right places

fix #215

Signed-off-by: Steven Zou <szou@vmware.com>

### CODE REVIEW

**Comment by steven-zou** [comment]
@holyhope 

Refreshed the README doc structure, please take a look?

notes: some of the linked docs are just placeholders.

**Comment by holyhope** [comment]
I am on it 👍

**Review by holyhope — COMMENTED** [review]
I reworked some documentation about the control loop :-)
Can you review it please?

File: README.md [hunk 1]
```diff
@@ -4,119 +4,137 @@
...
+
+## Getting started
+
+For a quick first try follow the instructions of this [tutorial](./docs/tutorial.md).
+
+## Versioning
+
+Versions of the underlying components are listed below:
+
+|  Components   |       Harbor      | MinIO operator | PostgreSQL operator | Redis operator |
```
[the remark this answers is not in the document]
Comment (steven-zou) [reply README.md:59]: yeah, a kustomization under `manifests/cluster`

**Comment by steven-zou** [comment]
This PR accumulated too many commits now. Let's merge it now and continue to refine and create docs with other new PRs.

cc @holyhope @heww @bitsf

**Review by heww — APPROVED** [review]

**Review by bitsf — APPROVED** [review]

**Pull request merged**